ios/FluentUI/Button/Button.swift

```diff
@@ -177,6 +188,9 @@ open class Button: UIButton, Shadowable, TokenizedControlInternal {
     public var ambientShadow: CALayer?
     public var keyShadow: CALayer?
 
+    /// Optional var to increase the default tappable size of the button.
+    public var tappableSize: CGSize?
```

Comment:
Should we make this objc as well?

Reply:
Yeah. Makes it a bit bigger, but no real reason to keep this in just Swift.